Modules/Core/Common/include/itkThreadPool.h

```diff
@@ -129,6 +126,9 @@ class ITKCommon_EXPORT ThreadPool : public Object
 
 private:
 
+  /** Only used to synchronize the global variable across static libraries.*/
+  itkGetGlobalDeclarationMacro(ThreadPoolGlobals, Pimpl);
```

Comment:
The inclusion of `itkThreadPool.h` was removed from `itkPyITKCommonCAPI.h`, so it needed to be added back somewhere. I added it through CMake so that it is called in the `xxxPython.cpp` module files.